### DISCUSSION TITLE
Plasma 6 Todo list before Release

### DISCUSSION THREAD

**Discussion by blackadderkate** [blackadderkate/weather-widget-2#173] [Announcements]
Just a Tracking list of things ToDo for the Plasma 6 Widget Release:

~~1.     Fixes for the widget when in the system tray.~~
~~2.     Fixes for Scaling / High DPI as suggested by @Whyhow314 .~~
3.     Implement fixes for when the user adds/deletes a location or changes the location order (which I think is the cause of @SpencerKersey 's bug).
~~4.     See if I can fix screen corruption when you change the order in the Places list.~~
5.     Create Screenshots of the new version
6.     Update Documentation/Wiki
7.     Release widget on KDE Widget Store

**Comment by punk0x29a** [comment]
Great to see you're working on this! Updated yesterday and I'm already missing the pressure plot (really helpful for meteoropaths when planning the day).

Cheers!
